frontend/src/components/Agreements/AgreementBLIAccordion/AgreementBLIAccordion.jsx

```diff
@@ -57,17 +70,28 @@ function AgreementBLIAccordion({
         ? updatedBudgetLines.filter((bli) => bli.status !== BLI_STATUS.DRAFT)
         : updatedBudgetLines;
     let budgetLinesForCards, subTotalForCards, feesForCards, totalsForCards;
+    let procurementShopAbbr = agreement.procurement_shop?.abbr;
 
     if (!isApprovePage || isDraftToPlanned) {
```

Comment:
this logic is getting had to follow, wonder if a case statement would make this more readable?